ui.tests/test-module/specs/formsportallinkcomponent.spec.js

```diff
@@ -0,0 +1,133 @@
+/*
+ *  Copyright 2021 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/// <reference types="cypress" />
+
+// This recipe is to test the AEM Forms Portal Link Component functionality
+
+// We are going to test:
+// 1. Link Component can be added and deleted
+// 2. Verify default values set in proxy component
+// 3. Check rendered link
+
+// Be sure to run the aem server
+// before running the tests below.
+
+const sitesSelectors = require('../libs/commons/sitesSelectors'),
+      afConstants = require('../libs/commons/formsConstants');
+
+describe('Link - Authoring', function () {
+    // we can use these values to log in
+    const   pagePath = "/content/core-components-examples/library/page-authoring/adaptive-form/link",
```

Comment:
Shouldn't the page path also change ? I think this test case would fail. Please check all occurrences.

Reply:
Updated test case